src/tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -617,6 +619,25 @@ public void ReplaceFileInApk(string file)
         SignApk(apkPath, apksigner);
     }
 
+    private static (string, string) GetZipToolWithArgs(string destZipPath)
```

Comment:
in my branch I replaced shelling out to `zip` with the `ZipFile.CreateFromDirectory` .NET API, curious why you went the PR route?